### PR TITLE
Add util for get model status

### PR DESCRIPTION
**Pull request by ynqa**
In tensorflow-serving-api, [get_model_status](https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/apis/BUILD#L236-L244) requires a generated code from status.proto in util.

```
>>> from tensorflow_serving.apis import get_model_status_pb2
...
ImportError: No module named util
```

### CODE REVIEW

File: tensorflow_serving/tools/pip_package/build_pip_package.sh [hunk 1]
```diff
@@ -44,9 +46,20 @@ function main() {
 
   cp bazel-serving/tensorflow_serving/apis/*_grpc.py \
     "${TMPDIR}/tensorflow_serving/apis"
+
```
Comment (gautamvasudevan) [inline tensorflow_serving/tools/pip_package/build_pip_package.sh]: Can you remove this extra space?

File: tensorflow_serving/tools/pip_package/build_pip_package.sh [hunk 2]
```diff
@@ -44,9 +46,20 @@ function main() {
 
   cp bazel-serving/tensorflow_serving/apis/*_grpc.py \
     "${TMPDIR}/tensorflow_serving/apis"
+  
+  cp bazel-genfiles/tensorflow_serving/util/*_pb2.py \
+    "${TMPDIR}/tensorflow_serving/util"
+
+  cp bazel-genfiles/external/org_tensorflow/tensorflow/core/lib/core/*.py \
```
Comment (gautamvasudevan) [inline tensorflow_serving/tools/pip_package/build_pip_package.sh]: What's being added from here?

File: tensorflow_serving/tools/pip_package/build_pip_package.sh (hunk 1 above)
Comment (ynqa) [reply tensorflow_serving/tools/pip_package/build_pip_package.sh]: Sorry, it's my mistake. I'll revert back it.

File: tensorflow_serving/tools/pip_package/build_pip_package.sh (hunk 2 above)
Comment (ynqa) [reply tensorflow_serving/tools/pip_package/build_pip_package.sh]: The status proto depends on tensorflow error_codes proto the same as py codes.

Ref:
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/apis/get_model_status.proto#L6
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/util/status.proto#L5
- https://github.com/tensorflow/tensorflow/blob/v1.9.0/tensorflow/core/lib/core/error_codes.proto

Comment (ynqa) [reply tensorflow_serving/tools/pip_package/build_pip_package.sh]: @gautamvasudevan The status proto depends on tensorflow error_codes proto the same as py codes.

Ref:
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/apis/get_model_status.proto#L6
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/util/status.proto#L5
- https://github.com/tensorflow/tensorflow/blob/v1.9.0/tensorflow/core/lib/core/error_codes.proto

File: tensorflow_serving/tools/pip_package/build_pip_package.sh (hunk 1 above)
Comment (ynqa) [reply tensorflow_serving/tools/pip_package/build_pip_package.sh]: @gautamvasudevan Sorry, it's my mistake. I'll revert back it.

Comment (ynqa) [reply tensorflow_serving/tools/pip_package/build_pip_package.sh]: @gautamvasudevan Done. Is it okay?

Comment (gautamvasudevan) [reply tensorflow_serving/tools/pip_package/build_pip_package.sh]: Hmm, I don't see a new commit with this change. Just the original 2.

File: tensorflow_serving/tools/pip_package/build_pip_package.sh (hunk 2 above)
Comment (gautamvasudevan) [reply tensorflow_serving/tools/pip_package/build_pip_package.sh]: Sounds good.

**Pull request merged**